thatkitebot/cogs/electronics.py

```diff
@@ -137,6 +138,8 @@ def calculate_lm317(b):
     r1 = si_prefix.si_parse(b["r1"])
     r2 = si_prefix.si_parse(b["r2"])
     vout = si_prefix.si_format(1.25 * (1 + (r2/r1)))
+    if vin - vout > 40 or vin - vout < 3:
```

Comment:
This is the code for the voltage divider. Do you really want to add a check for that? Wouldn't it make more sense to add this to the lm317 code?